pkgs/development/interpreters/boron/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, stdenv
+, fetchgit
+, zlib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "boron";
+  version = "2.0.8";
+
+  src = fetchgit {
+    url = "https://git.code.sf.net/p/urlan/boron/code";
+    rev = "v${version}";
+    sha256 = "sha256-RpnMs9DqcbK/gLDH5x3/DvxodJb4tR76Yk8SbYtIkV4=";
+  };
+
+  # this is not a standard `configure`
+  configurePhase = ''
+    patchShebangs .
+    ./configure
+  '';
+
+  makeFlags = [ "DESTDIR=$(out)" ];
+
+  buildInputs = [
+    zlib
+  ];
+
+  installTargets = "install install-dev";
+
+  meta = with lib; {
+    homepage = "http://urlan.sourceforge.net/boron/";
+    description = "Scripting language and C library useful for building DSLs";
+    license = licenses.gpl3Plus;
```

Comment:
The license is actually not GPLv3+ but LGPLv3, as stated in the README, totally missed the first time.

Reply:
Fixed in https://github.com/NixOS/nixpkgs/pull/180865/commits/e2bfd1d1c2e3dae1e6b05a50e47322cd3abeecfa